src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/login.yaml

```
@@ -1,47 +1,52 @@
 sylius_twig_hooks:
     hooks:
         'sylius_shop.account.login':
-            form:
-                template: '@SyliusShop/account/login/form.html.twig'
-                priority: 0
+            content:
+                template: '@SyliusShop/account/login/content.html.twig'
+                priority: 0        
 
-        'sylius_shop.account.login.form':
+        'sylius_shop.account.login.content':
             login_container:
-                template: '@SyliusShop/account/login/form/login_container.html.twig'
+                template: '@SyliusShop/account/login/content/login_container.html.twig'
                 priority: 100
             register_container:
-                template: '@SyliusShop/account/login/form/register_container.html.twig'
+                template: '@SyliusShop/account/login/content/register_container.html.twig'
                 priority: 0
 
-        'sylius_shop.account.login.form.login_container':
+        'sylius_shop.account.login.content.login_container':
             header:
-                template: "@SyliusShop/account/login/form/login_container/header.html.twig"
-                priority: 400
+                template: "@SyliusShop/account/login/content/login_container/header.html.twig"
```

Not sure which syntax `'` or `"` is official, but file uses both.

```suggestion
                template: '@SyliusShop/account/login/content/login_container/header.html.twig'
```